package-lock.json

```diff
@@ -954,12 +973,26 @@
       }
     },
     "elm": {
-      "version": "0.19.0-bugfix6",
-      "resolved": "https://registry.npmjs.org/elm/-/elm-0.19.0-bugfix6.tgz",
-      "integrity": "sha512-nD/oK/nG26ULh94GREsILA9aQa3Gon+KKBmS0Z0MstywLWwrhuWKba8gfiIWLdnpQWHFIf+UsThk6Z71UWi6TQ==",
+      "version": "0.19.0",
```

Comment:
You went backwards on the elm version -- This should be "0.19.0-no-deps" for the latest version

Reply:
Changed to the suggested string in `package.json` and ran `npm install`.